packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/quanticDocumentSuggestion.css

```diff
@@ -0,0 +1,21 @@
+.excerpt {
+  font-size: 14px;
+}
+.loading-holder {
+  position: relative;
+  height: 3.5rem;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.slds-card {
+  border: 1px solid #dddbda;
```

Comment:
to try to avoid hard coded styling have you tried using some of the GA design tokens for border colors?